triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -112,6 +115,24 @@ class Server(dar: Option[Dar[(PackageId, Package)]], jdbcConfig: Option[JdbcConf
   private def listRunningTriggers(jwt: Jwt): List[String] = {
     triggersByToken.getOrElse(jwt, Set()).map(_.toString).toList
   }
+
+  private def timeStamp(): String = {
+    DateTimeFormatter.ofPattern("yyyy-MM-dd_HH:mmss").format(LocalDateTime.now)
+  }
+
+  private def triggerStatus(t: RunningTrigger, msg: String): Unit = {
```

Comment:
A verb name would be more accurate here, like `logTriggerStatus`.

Reply:
Right on.